### DISCUSSION TITLE
[JSS Angular Upgrade to 18]

### DISCUSSION THREAD

**Discussion by YogiReddyHS** [Sitecore/jss#1974] [Help] [answered]
### Description

The latest JSS SDK supports angular version 17.3, while it is going end of life support in May 2025. Is there a roadmap to upgrade to Angular 18 or higher? Is there a policy around JS framework upgrade support.

**Comment by illiakovalenko** [comment]
@YogiReddyHS Hey! We understand the importance of staying aligned with the latest Angular versions, especially as older versions approach their end-of-life support. Ensuring compatibility with the latest JS frameworks is something we actively consider as part of our ongoing roadmap discussions for JSS.
 
While we cannot share specific timelines or commitments at this moment, please know that upgrading to Angular 18 or higher is aligned with our long-term vision for JSS. Our focus remains on providing developers with modern, flexible, and forward-compatible tools, and framework updates are an important part of that mission.
 
Thanks for the valuable feedback!

**Comment by illiakovalenko** [comment, marked as answer]
@YogiReddyHS The upgrade to Angular 15 was released as part of JSS 22.5.0. Feel free to upgrade!
